index.html

```diff
@@ -1,69 +1,269 @@
+                  <li class="work__item">Back End Dev</li>
+                  <li class="work__item">2015</li>
+                </ul>
+              </div>
+              <p class="portfolio__description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="portfolio__tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+              <div>
+                <a class="button_primary" href="#">See project</a>
+              </div>
+            </div>
+          </article>
+        </div>
+      </section>
+      <section class="section section_white" id="about_me">
+          <div class="about">
+            <h2 class="about_title">About Myself</h2>
+            <p class="color-info">
+              Hello I'm a software developer! I can help you build a product,
+              feature or website. Look through some of my work and experience!
+              If you like what you see and have a project you need coded,
+              don't hesitate to contact me.
+            </p>
+            <h2 class="connect">LET’S CONNECT</h2>
+            <ul class="social_icons">
+              <li><a href=""><img src="assets/twitter.png" alt="Twitter" /></a></li>
+              <li><a href=""><img src="assets/Linkedin.png" alt="LinkedIn" /></a></li>
+              <li><a href=""><img src="assets/medium.png" alt="Medium" /></a></li>
+              <li><a href=""><img src="assets/github.png" alt="Github" /></a></li>
+              <li><a href=""><img src="assets/angel.png" alt="Angel" /></a></li>
+            
+            </ul>
+            </div>
+            <div>
+              <a  class="button_primary">Get my resume</a>
+            </div>
+            <div class="category-about_page">
+              <header
+                class="about_page_header flex-row justify-between align-center"
+              >
+                <h3 class="about_page_subtitle">Languages</h3>
+                <img src="assets/arr_d.svg" alt="Arrow down icon" />
+              </header>
+              <ul class="about_page_list">
+                <li class="about_page__item">
+                  <img
+                    src="assets/Ellipse1.png"
+                    alt="javascript image"
+                  />
+                  JavaScript
+                </li>
+                <li class="about_page__item">
+                  <img
+                    src="assets/Ellipse2.png"
+                    alt="html image"
+                  />
+                  HTML
+                </li>
+                <li class="about_page__item">
+                  <img
+                    src="assets/Ellipse3.png"
+                    alt="css image"
+                  />
+                  CSS
+                </li>
+              </ul>
+            </div>
```

Comment:
- [ ] ***[OPTIONAL]*** `Though it is optional it is a very important bug that you need to fix.` 
- Kindly consider adjusting the alignment of some of the features of this section which I've highlighted in the image below to have a replica of what's provided on the Figma design. You'll agree with me it does not give a better user experience 👍

- ***Your Design***
- ![image](https://github.com/montsecruz9/personal-portfolio/assets/60083437/98f06674-8eca-466a-bbd1-c73a80afc6b7)
- ***Figma Design***
- ![image](https://github.com/montsecruz9/personal-portfolio/assets/60083437/4463e4fd-44df-4643-9e9a-71b7cbc22cb3)

Reply:
Thanks @Tufahel !! But i had that fixed already before. See: 
![image](https://github.com/montsecruz9/personal-portfolio/assets/104812189/11d96db5-6bf7-402a-bf38-130695cfad9d)